### DISCUSSION TITLE
答题

### DISCUSSION THREAD

**Discussion by tt619252615** [Fatpandac/fuck_cqooc#53] [Ideas]
老哥，我看你封装的这么好。我感觉可以把答题也完成。建一个公共数据库，把不同课程答题如果做答题了上传。答题可以使用使用百度云的文心一言api。我测试答题效果还可以。

**Comment by RX-105** [comment]
感谢你提出的idea！
关于答题的功能，我们曾经有过这种设想，计划是通过网站api和用户答案收集来实现这么一个数据库，我认为有一定的可行性，所以未来可能会实现这个功能。
至于大模型答题的质量无法保证，可行性有待商榷，所以目前不会考虑。

**Reply by tt619252615** [reply]
其实我个人认为实现一个数据库比较现实，我也在你的api构建之上设计了文心一言的答题，效果不是那么好，应该是我测试的模型不怎么样。不过总的来说比我自己做要好点，回复的答案基本是对的。

**Reply by Fatpandac** [reply]
我有找到一个会返回小节测试答案的 API 一直想弄的但是没有时间，如果你对这个感兴趣想动手尝试的话我很乐意整理相关 API 的资料给你，否则只能我有时间了再慢慢实现了。
